.github/workflows/lint.yml

```diff
@@ -0,0 +1,126 @@
+# Copyright 2020 Vectorized, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+name: Lint all codez
+on:
+  push:
+    tags: 'release-*'
+    branches:
+      - dev
+      - release
+  pull_request:
+    branches:
+      - main
+      - dev
+
+jobs:
+  go:
+    name: Lint go files
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Check out code
+      uses: actions/checkout@v2
+
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.13.6
+      id: go
+
+    - name: install crlfmt
+      run: |
+        # Workaround to download the specific version we use internally
+        cat > go.mod << EOF
+        module vectorized
+        require (
+          github.com/cockroachdb/crlfmt v0.0.0-20200309141329-97b747f03b21
+        )
+        go 1.13
+
+        EOF
+
+        go mod download
+        go install github.com/cockroachdb/crlfmt
+        rm -rf go.mod
+
+    - name: lint go files
+      run: |
+        find . -name *.go -type f | xargs -n1 crlfmt -wrap=80 -w
+        git diff --exit-code
+
+  js:
+    name: Lint js files
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Check out code
+      uses: actions/checkout@v2
+
+    - name: Set up Node
+      uses: actions/setup-node@v2-beta
+      with:
+        node-version: '12.16.1'
+
+    - name: Set up prettier
+      run: npm install prettier
+
+    - name: Lint JS files
+      run: npx prettier --list-different src/**/*.{ts,js}
+
+  py:
+    name: Lint python files
+    runs-on: ubuntu-20.04
+    steps:
+
+    - name: Check out code
+      uses: actions/checkout@v2
+
+    - name: Set up python
+      run: |
+        sudo apt-get install -y python3 python3-pip
+        sudo pip install yapf
+
+    - name: lint python files
+      run: find . -name *.py -type f | xargs -n1 yapf -d
+
+  cpp:
+    name: Lint files with clang-format
+    runs-on: ubuntu-latest
+    # Use ubuntu:groovy explicitly as it is the first version to have
+    # clang-format 11 available
+    container: ubuntu:groovy
```

Comment:
Is `clang-10` noticeably different for the purpose of linting? No big deal either way.

Reply:
I don't think there would me that many differences but it's good to be consistent.
Since pinning to a specific container image is a low price to pay for that :)